cmds/boot/systemboot/main.go

```diff
@@ -18,6 +19,100 @@ import (
 	"github.com/u-root/u-root/pkg/smbios"
 )
 
+const (
+	_FB_OEM_IPMI_NET_FUNCTION = 0x36
+
+	_FB_OEM_SET_PROC_INFO = 0x10
+	_FB_OEM_SET_DIMM_INFO = 0x12
+
+	_WIWYNN_ID0 = 0x0
+	_WIWYNN_ID1 = 0x9c
+	_WIWYNN_ID2 = 0x9c
+)
+
+type oemProcProductNameDataParam1 struct {
+	ManufacturerID    [3]uint8
+	Index             uint8
+	ParameterSelector uint8
+	ProductName       [48]byte
+}
+
+func (ProccessorDataParam1 *oemProcProductNameDataParam1) Data() {}
+
+type oemProcBasicInfoDataParam2 struct {
```

Comment:
Is `cmds/boot` the best place to define all these structs?

Reply:
How about creating a file named oem.go in IPMI package? We can define all OEM stuffs in the file.

Reply:
Sounds good

Reply:
I created a file named oem.go for implementing all IPMI OEM features.